commons/src/main/com/mbrlabs/mundus/commons/assets/meta/MetaLoader.java

```diff
@@ -57,6 +57,7 @@ private void parseTerrain(Meta meta, JsonValue jsonTerrain) {
 
         final MetaTerrain terrain = new MetaTerrain();
         terrain.setSize(jsonTerrain.getInt(MetaTerrain.JSON_SIZE));
+        terrain.setUv(jsonTerrain.getFloat(MetaTerrain.JSON_UV_SCALE, 60));
```

Comment:
I think the value 60 would be better in a constant file as a constant and you can use it in every place. What do you think?

Reply:
hmm yeah, could use it there and in the uv vector initialization, trying to think if mundus already has a constants file

Reply:
Could put is as a public static in Terrain class actually. It already has some other default values there

Reply:
done

Reply:
Nice :) Thanks